src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/Orderly.kt

```diff
@@ -270,6 +270,18 @@ class Orderly(val isReviewer: Boolean,
         }
     }
 
+    override fun getAllReportTags(): Map<String, List<String>>
```

Comment:
For the index page, provide the report tags separately to the versions returned. We don't check here whether the user has permission to access the report since the tags will only be appended by the view model builder for reports where versions have been confirmed accessible by the user.